packages/v2ray/0001-fix-config-pathes.patch

```diff
@@ -0,0 +1,132 @@
+--- a/go/pkg/mod/github.com/adrg/xdg@v0.4.0/paths_unix.go	2022-09-24 18:18:29.907660821 +0800
++++ b/go/pkg/mod/github.com/adrg/xdg@v0.4.0/paths_unix.go	2022-09-24 18:34:47.217660448 +0800
+@@ -27,27 +27,27 @@
+ func initBaseDirs(home string) {
+ 	// Initialize standard directories.
+ 	baseDirs.dataHome = xdgPath(envDataHome, filepath.Join(home, ".local", "share"))
+-	baseDirs.data = xdgPaths(envDataDirs, "/usr/local/share", "/usr/share")
++	baseDirs.data = xdgPaths(envDataDirs, "@TERMUX_PREFIX@/local/share", "@TERMUX_PREFIX@/share")
+ 	baseDirs.configHome = xdgPath(envConfigHome, filepath.Join(home, ".config"))
+ 	baseDirs.config = xdgPaths(envConfigDirs, "/etc/xdg")
+ 	baseDirs.stateHome = xdgPath(envStateHome, filepath.Join(home, ".local", "state"))
+ 	baseDirs.cacheHome = xdgPath(envCacheHome, filepath.Join(home, ".cache"))
+-	baseDirs.runtime = xdgPath(envRuntimeDir, filepath.Join("/run/user", strconv.Itoa(os.Getuid())))
++	baseDirs.runtime = xdgPath(envRuntimeDir, filepath.Join("@TERMUX_PREFI@/var/run/user", strconv.Itoa(os.Getuid())))
```

Comment:
Since termux is a single user environment we can use just `baseDirs.runtime = xdgPath(envRuntimeDir, "@TERMUX_PREFIX@/var/run")`

(also note typo PREFI -> PREFIX)